### PR TITLE
Route not found page added

### PR DESCRIPTION
**Pull request by Nikhilesh002**
## Related Issue
None

## Description
- RouteNotFound page added
- Bootstrap link updated
- Removed bootsteap cdn and then installed via package manager

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): _____Bootstrap link updated_____

## Screenshots / videos (if applicable)
image

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
- Added RouteNotFound page.
- Updated bootsteap cdn link from 4.3.1 to 5.3.3 version
- removed bootsteap cdn and then installed via package manager

### CODE REVIEW

**Comment by Nikhilesh002** [comment]
Fixes #138 

@Mansi168 please review my PR and merge it if its good to go, with gssoc and revelant level labels

**Comment by Nikhilesh002** [comment]
Hi @Mansi168 , please approve my pr

**Comment by Nikhilesh002** [comment]
@Mansi168  pls reply

**Comment by Nikhilesh002** [comment]
@everly-gif  please look this pr

File: package.json [hunk 1]
```diff
@@ -24,11 +24,12 @@
     "loader-utils": "^3.2.1",
     "mdb-react-ui-kit": "^8.0.0",
     "nextjs-websocket": "^1.0.11",
+    "pkg.json": "^2.0.8",
```
Comment (everly-gif) [inline package.json:27]: @Nikhilesh002 what is the need for this?

**Review by everly-gif — APPROVED** [review]
Looks good to me @Nikhilesh002 . I tested it out and really nice to see you made it responsive as well. Great work :) . I'm approving these. @Mansi168 please have a look as well

File: package.json (hunk 1 above)
Comment (Nikhilesh002) [reply package.json:27]: I dont know, it was suggested in terminal, then I installed it.

**Pull request merged**